app/views/items/show.html.erb

```diff
@@ -1,26 +1,27 @@
 <% content_for :sidebar do %>
-  <%= link_to 'Return to Listing', items_path, :class => 'btn' %></br></br>
-  <%= edit_item_button %></br></br>
-  <%= add_repair_button %></br></br>
-  <%= add_inspection_button %></br></br>
+  <%= link_to 'Return to Listing', items_path, class: 'btn btn-primary', style: 'width: 100%' %><br/><br/>
+  <%= edit_item_button %><br/><br/>
+  <%= add_repair_button %><br/><br/>
+  <%= add_inspection_button %><br/><br/>
```

Comment:
I think you are more right than I was, but perhaps no slash would be best....
http://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br